bundles/automation/org.eclipse.smarthome.automation.rest/src/main/java/org/eclipse/smarthome/automation/rest/internal/ModuleTypeResource.java

```diff
@@ -73,6 +80,15 @@ protected void unsetModuleTypeRegistry(ModuleTypeRegistry moduleTypeRegistry) {
         this.moduleTypeRegistry = null;
     }
 
+    @Reference(cardinality=ReferenceCardinality.MANDATORY, policy=ReferencePolicy.STATIC)
```

Comment:
`cardinality=ReferenceCardinality.MANDATORY, policy=ReferencePolicy.STATIC` is default, so no need to set it explicit.
Can you remove it here and on other places